src/cljam/io/bam/decoder.clj

```diff
@@ -124,7 +124,10 @@
          cigar-bytes (lsb/read-bytes buffer (* n-cigar-op 4))
          [cigar ^long len] (cigar/decode-cigar-and-ref-length cigar-bytes)
          ref-end     (int ^long (if (zero? len) pos (dec (+ pos len))))
-         seq         (decode-seq (lsb/read-bytes buffer (quot (inc l-seq) 2)) l-seq)
+         seq         (if (= l-seq 0)
```

Comment:
```suggestion
         seq         (if (zero? l-seq)
```

BTW, I thought the annotation at the top should actually point to this line since the hunk is contained in this PR. Would you take a look at `clj-lint-action`?

<img width="759" alt="image" src="https://user-images.githubusercontent.com/5086013/122368051-f6a5bb80-cf97-11eb-9c7b-30a22d56dfb8.png">